services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java

```diff
@@ -44,27 +46,30 @@
 @Api(tags = "v1/news", value = "v1/news", description = "Managing news")
 public class NewsRestResourcesV1 implements ResourceContainer, Startable {
 
-  private static final Log       LOG                             = ExoLogger.getLogger(NewsRestResourcesV1.class);
+  private static final Log         LOG                             = ExoLogger.getLogger(NewsRestResourcesV1.class);
 
-  private static final String    PUBLISHER_MEMBERSHIP_NAME       = "publisher";
+  private static final String      PUBLISHER_MEMBERSHIP_NAME       = "publisher";
 
-  private final static String    PLATFORM_WEB_CONTRIBUTORS_GROUP = "/platform/web-contributors";
+  private final static String      PLATFORM_WEB_CONTRIBUTORS_GROUP = "/platform/web-contributors";
 
-  //private final static String    PLATFORM_ADMINISTRATORS_GROUP = "/platform/administrators";
+  // private final static String PLATFORM_ADMINISTRATORS_GROUP =
+  // "/platform/administrators";
```

Comment:
Clean code please!

Reply:
It's linked to @azayati refactoring. I just did a code formatting for the whole file code because I modified in each function.

Reply:
the comments will be cleaned later in an additional later refactoring PR

Reply:
good catch!